website/siteConfig.js

```diff
@@ -65,11 +66,11 @@ const siteConfig = {
   separateCss: [
     "css/static_website.css"
   ],
-  // algolia: {
-  //   apiKey: '',
-  //   indexName: '',
-  //   algoliaOptions: {} // Optional, if provided by Algolia
-  // },
+  algolia: {
+    apiKey: 'f7e83f64ca037b5315b5ccd56f79910c',
```

Comment:
is it recommended by algolia to add this key here?

Reply:
https://docusaurus.io/docs/en/search. In this section of the documentation, docusaurus give no warnings about the exposition of the api key.